packages/carbon-web-components/src/components/copy/copy.ts

```diff
@@ -17,6 +17,8 @@ import CDSIconButton from '../icon-button/icon-button';
  * Copy.
  *
  * @element cds-copy
+ *
+ * @csspart content - The content. Usage `cds-copy::part(content)`
```

Comment:
Sorry to nit pick. I think it will be less ambiguous to call it `tooltip-content`. If I think about what the copy button's `content` is, I might not end up thinking that this is actually referring to the tooltip content.
```suggestion
 * @csspart tooltip-content - The content. Usage `cds-copy::part(tooltip-content)`
```